pom.xml

```diff
@@ -291,6 +291,13 @@
                             <pattern>org.yaml.snakeyaml</pattern>
                             <shadedPattern>net.datafaker.shaded.snakeyaml</shadedPattern>
                         </relocation>
+                        <!-- Manually relocate shaded snakeyaml multi-release classes to the correct package. This is
+                             needed due to a bug in the Maven Shade plugin, which only relocates the content but not
+                             the package structure itself, see https://issues.apache.org/jira/browse/MSHADE-406. -->
+                        <relocation>
+                            <pattern>META-INF/versions/9/org.yaml.snakeyaml</pattern>
+                            <shadedPattern>META-INF/versions/9/net.datafaker.shaded.snakeyaml</shadedPattern>
```

Comment:
I'm curious whether there is a more generic solution which could handle other versions once they appear...
Otherwise any new version will lead to the same issue

Reply:
There might be an (undocumented way) to do this, I am currently looking at the Maven plugin code.

Reply:
The follwing commit should address your concern. Now all versions of the `org.yaml.snakeyaml` package are being relocated. Although new versions will not come around very often and I hope that the actual bug in the Maven plugin will get fixed sooner or later so that this workaround can be removed.